systest/multi-tenancy/integration_basic_helper_test.go

```diff
@@ -0,0 +1,234 @@
+//go:build integration
+// +build integration
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and 
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dgraph-io/dgraph/graphql/e2e/common"
+	"github.com/dgraph-io/dgraph/testutil"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+type liveOpts struct {
+	rdfs      string
+	schema    string
+	gqlSchema string
+	creds     *testutil.LoginParams
+	forceNs   int64
+}
+
+func (suite *MultitenancyTestSuite) postGqlSchema(schema string, accessJwt string) {
+	t := suite.T()
+	groupOneHTTP := testutil.ContainerAddr("alpha1", 8080)
+	header := http.Header{}
+	header.Set("X-Dgraph-AccessToken", accessJwt)
+	common.SafelyUpdateGQLSchema(t, groupOneHTTP, schema, header)
+}
+
+func (suite *MultitenancyTestSuite) liveLoadData(opts *liveOpts) error {
+	t := suite.T()
+	dir := t.TempDir()
+	rdfFile := filepath.Join(dir, "rdfs.rdf")
+	require.NoError(t, os.WriteFile(rdfFile, []byte(opts.rdfs), 0644))
+	schemaFile := filepath.Join(dir, "schema.txt")
+	require.NoError(t, os.WriteFile(schemaFile, []byte(opts.schema), 0644))
+	gqlSchemaFile := filepath.Join(dir, "gql_schema.txt")
+	require.NoError(t, os.WriteFile(gqlSchemaFile, []byte(opts.gqlSchema), 0644))
+	// Load the data.
+	return testutil.LiveLoad(testutil.LiveOpts{
```

Comment:
we can also move the live loader code into dgraphtest, you could do it as a separate PR too